Tests/MeshLib/TestMeshNodeSearch.cpp

```diff
@@ -60,11 +64,43 @@ class MeshLibMeshNodeSearchInSimpleHexMesh : public testing::Test
 	Mesh* _hex_mesh;
 };
 
+TEST_F(MeshLibMeshNodeSearchInSimpleQuadMesh, PointSearch2)
```

Comment:
Please rename PointSearch2 to something more describing, or at least add a comment, what is tested here.